README.md

```diff
@@ -17,10 +17,19 @@ Katkıda bulunmak için lütfen [Discord Grubumuz'dan](https://discord.gg/cyN3vr
 
 ## Çeviri Kuralları
 
+Çeviri yapmaya başlamadan önce lütfen [kavramlar dizinini](https://github.com/solidity-docs/tr-turkish/blob/develop/GLOSSARY.md) kontrol ediniz.
```

Comment:
Sadece ufak bir not: relative bağlantı kullanırsan diğer branchler'de de repo içi linkler çalışır: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files

Reply:
Tamamdır hallediyorum.